flow-tests/test-theme-editor/pom.xml

```
@@ -15,7 +15,7 @@
     <properties>
         <maven.deploy.skip>true</maven.deploy.skip>
         <vaadin.devmode.liveReload.enabled>true</vaadin.devmode.liveReload.enabled>
-        <vaadin.components.version>24.3-SNAPSHOT</vaadin.components.version>
+        <vaadin.components.version>24.4-SNAPSHOT</vaadin.components.version>
```

There is no flow-components 24.4-SNAPSHOT yet. We should probably stay with 24.3-SNAPSHOT
```suggestion
        <vaadin.components.version>24.3-SNAPSHOT</vaadin.components.version>
```